frontend/src/Components/Form/UMaskInput.js

```diff
@@ -1,33 +1,44 @@
 /* eslint-disable no-bitwise */
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
+import translate from 'Utilities/String/translate';
 import EnhancedSelectInput from './EnhancedSelectInput';
 import styles from './UMaskInput.css';
 
 const umaskOptions = [
   {
     key: '755',
-    value: '755 - Owner write, Everyone else read',
+    get value() {
+      return `755 - ${translate('Umask755Description')}`;
```

Comment:
I think we could include the octal in the translation for added context, but not sure if that would have a downside for some languages?

Reply:
I chose not to here, to be sure that people didn't translate it incorrectly. I could always pass it in as a translation variable?

Reply:
Yeah, makes sense, I think a variable would be good though